tool-populate/index.js

```diff
@@ -104,24 +104,44 @@ function destroyObjects(objects) {
   }
 }
 
+function setRoleACLs() {
```

Comment:
My bad. Won't we need to give admin write access though? So maybe I should modify the helper function—remove setting the public rights?